### PR TITLE
Update autocomplete guide

### PR DESCRIPTION
**Pull request by mthuret**
Mention `connectAutoComplete`. 
Mention new @mentions examples.

### CODE REVIEW

File: docgen/src/guide/Autocomplete_menu.md [hunk 1]
```diff
@@ -6,17 +6,17 @@ category: guide
 navWeight: 20
 ---
 
-You can build with React InstantSearch an autocomplete menu that uses an external autocomplete component.
+You can build with React InstantSearch an autocomplete menu that uses an external autocomplete component. It takes advantage of the [`connectAutoComplete`](connectors/connectAutoComplete.html) connector.
```
Comment (Haroenv) [inline docgen/src/guide/Autocomplete_menu.md]:
```diff
+ You can build an autocomplete menu with React InstantSearch 
- You can build with React InstantSearch an autocomplete menu
```

File: docgen/src/guide/Autocomplete_menu.md [hunk 2]
```diff
@@ -6,17 +6,17 @@ category: guide
...
+ showing you how to build an autocomplete menu that will let you:
 * display hits from different indices
+* have a textarea able to propose suggestions like mentions. 
 
 Those examples use advanced concepts from React InstantSearch. If you're not familiar with
 them you can read their guide first:
 
-* [Custom connectors](guide/Custom_connectors.html)
+* [Connectors](guide/Connectors.html)
 * [Virtual widgets](guide/Virtual_widgets.html)
```
Comment (Haroenv) [inline docgen/src/guide/Autocomplete_menu.md]: what is this link still needed for? 😄

Comment (mthuret) [inline docgen/src/guide/Autocomplete_menu.md]: Well, `connectAutoComplete` is a connector? Before it, you needed to use custom connectors.

Comment (Haroenv) [inline docgen/src/guide/Autocomplete_menu.md]: I'm talking about the link to "Virtual Widgets"

**Pull request merged**